### ISSUE TITLE
MvNormal create using chol could not be shown in notedook

### ISSUE THREAD

**Issue by Shimuuar** [pymc-devs/pymc#3450]
## Description of your problem

If one creates MvNormal distribution using chol for describing covariance it could be shown in notebook. Other methods work fine. Here is example

```python
import pymc3 as pm
import numpy as np

with pm.Model() as model:
    d = pm.MvNormal('x', mu=np.zeros(2), chol=np.eye(2), shape=2)
    print(d.distribution._repr_latex_())
```

**Please provide the full traceback.**
```python
Traceback (most recent call last):
  File "fail.py", line 9, in <module>
    print(d.distribution._repr_latex_())
  File "/nix/store/4c6ihiawh232fszikcyxhdk32rzk4l28-python3-3.7.2-env/lib/python3.7/site-packages/pymc3/distributions/multivariate.py", line 286, in _repr_latex_
    .format(name, name_mu, self._repr_cov_params(dist)))
  File "/nix/store/4c6ihiawh232fszikcyxhdk32rzk4l28-python3-3.7.2-env/lib/python3.7/site-packages/pymc3/distributions/multivariate.py", line 145, in _repr_cov_params
    chol = get_variable_name(self.chol)
AttributeError: 'MvNormal' object has no attribute 'chol'
```

It looks like `__init__` does not set `chol` field which function `_repr_cov_params` expects. By looking at source code problem persists in master

## Versions and main components

* PyMC3 Version: 3.6
* Theano Version:
* Python Version: 3.7
* Operating system: linux
* How did you install PyMC3: (conda/pip) nix

**Comment by chang111** [comment]
Thank You for your proposal. I will try to fix it!

**Comment by alfredaita** [comment]
PYMC3 relies on theano, which is no longer being developed. Theano will not run on python 3.6 or above.
So how could PYMC3 work.

**Comment by fonnesbeck** [comment]
Theano absolutely runs on Py 3.6+.

**Comment by Dpananos** [comment]
Do we know what is causing the issue?  I can try and fix it this week.

**Comment by lucianopaz** [comment]
@Dpananos, it's being stored as [`chol_cov`](https://github.com/pymc-devs/pymc3/blob/master/pymc3/distributions/multivariate.py#L79) instead of `chol`. This should be fixed consistently.

**Comment by Dpananos** [comment]
@lucianopaz So is this as easy as doing a `git grep` or are there other considerations to making the fix?

**Comment by lucianopaz** [comment]
I wouldn't trust a simple grep replace. I recommend that you read through the two involved classes and try to polish out the inconsistent names used for `chol`.

**Comment by Dpananos** [comment]
Done.  I'll get to it this week. Are there any tests written for `__repr_latex()`?

**Comment by lucianopaz** [comment]
Almost none, that's one of the main points that lower our code coverage... If you would be willing to write tests for `__repr_latex`, they would be great and hugely benefit the coverage!

**Comment by michaelosthege** [comment]
Closed by #3472
